include/Brig.h

```diff
@@ -0,0 +1,1131 @@
+// University of Illinois/NCSA

```

Comment:
I am ok with moving this file and other files from src/inc/ to include.

But overall, this patch series is not removing the files from the other directories. Same applies to the other files below.